src/components/bottom_bar/bottom_bar.ts

```diff
@@ -88,7 +81,7 @@ export class BottomBar extends Component<Props, SpreadsheetChildEnv> {
   private bottomBarRef = useRef("bottomBar");
   private sheetListRef = useRef("sheetList");
 
-  private dndHelper: DOMDndHelper | undefined;
+  private dndHook = useDOMDnd();
```

Comment:
the word "hook" is pretty useless here. I'd name this just `dnd` or `dragAndDrop`.
```suggestion
  private dragAndDrop = useDOMDnd();
```

Reply:
You think ? I'd get a bit confused at what the hell a `dragAndDrop ` object is. Is it s state ? A boolean ? A helper ? Not that `dndHook` is that much clearer, but at least I'd get that it's a hook that does things related to drag & drop.

Reply:
Actually, the hook is the *function* `useDOMDnd`, not what it returns.
And what matters is that it's clear what's happening when reading. I think things like `dragAndDrop.start(...)` is quite self-explanatory.